core-web/libs/sdk/react/src/lib/next/hooks/useIsDevMode.ts

```diff
@@ -1,26 +1,24 @@
 import { useContext, useEffect, useState } from 'react';
 
 import { getUVEState } from '@dotcms/uve';
+import { DEVELOPMENT_MODE } from '@dotcms/uve/internal';
 import { UVE_MODE } from '@dotcms/uve/types';
 
-import { DotCMSPageRendererMode, DotCMSPageContext } from '../contexts/DotCMSPageContext';
+import { DotCMSPageContext } from '../contexts/DotCMSPageContext';
 
 /**
  * @internal
  * A React hook that determines if the current environment is in development mode.
  *
  * The hook returns `true` if either:
- *   - The context mode (or the optional `renderMode` argument) is set to 'development', or
- *   - The application is running inside the DotCMS editor (as determined by `isInsideEditor()`).
+ *   - The application is running inside the DotCMS editor (as determined by `getUVEState()`).
  *
- * @param {DotCMSPageRendererMode} [renderMode] - Optional override for the render mode.
  * @returns {boolean} - `true` if in development mode or inside the editor; otherwise, `false`.
  */
-export const useIsDevMode = (renderMode?: DotCMSPageRendererMode) => {
+export const useIsDevMode = (): boolean => {
```

Comment:
Not good naming so...

UVE now have 3 modes: draft (edit), preview and published (live).

When we say "is dev mode" is like we are in dev mode in our app, but what we are actually evaluating here is if we are in "edit" mode of the UVE.